src/ja.json

```diff
@@ -0,0 +1,217 @@
+      "dont-ask-again" : "今後表示しない",
+      "launch-game" : "ゲーム起動",
+      "quit" : "終了",
+      "rating-very-poor" : "非常に悪い",
+      "rating-poor" : "悪い",
+      "rating-fair" : "まぁまぁ",
+      "rating-good" : "良い",
+      "rating-great" : "とても良い",
+      "back-to-hosts" : "ホストに戻る",
+      "rainway" : "Rainway",
+      "looking-for-service-games" : "{{service}ゲームを探しています...",
+      "looking-for-games" : "ゲームを探す",
+      "scanning-for-games-message" : "このシステムにインストールされているゲームをスキャンしています",
+      "could-not-find-games" : "このシステムにインストールされているゲームが見つかりませんでした",
+      "think-its-a-mistake-reload" : "これは間違いですか？ <1> 今すぐ再ロード </1>  .",
+      "audio-codec-not-supported" : "オーディオコーデックがサポートされていません",
+      "video-codec-not-supported" : "ビデオコーデックがサポートされていません",
+      "gameplay-resuming" : "ゲームプレイを再開",
+      "network-connection-lost" : "ネットワーク接続が失われました",
+      "removed-token-alert" : "トークンをデータベースから削除しました。<1>このページ</1>にアクセスして、プロバイダからのアクセス許可を取り消してください。",
+      "session-expired" : "セッションが期限切れです",
+      "finding-your-games" : "ゲームを見つける",
+      "no-games-found" : "ゲームが見つかりません",
+      "really-log-out" : "本当にログアウトしますか?",
+      "alert" : "アラート",
+      "accept" : "承諾",
+      "decline" : "拒否",
+      "installed-games" : "インストール済みのゲーム",
+      "time-played" : "プレイ時間",
+      "watch-out-cpu-message" : "気を付けて！ エンコードにCPUを使用すると、ストリームのパフォーマンスに影響する可能性があります。",
+      "software-uses-cpu" : "ソフトウェア(CPUを使用)",
+      "unable-to-connect" : "接続できません",
+      "trying-to-connect-from-host-message" : "Rainwayダッシュボードをホストしているコンピューターから接続しようとしています。別のデバイスからWebクライアントに移動してみてください。トレイからダッシュボードを開いて、ゲームをローカルで起動することもできます。",
+      "cant-connect-for-some-reason-message" : "Rainwayダッシュボードが見つかりましたが、何らかの理由で接続できません。詳細については、以下を参照してください。",
+      "several-devices-running-cant-connect-message" : "複数のデバイスでRainwayダッシュボードを実行していますが接続できません。詳細については、以下を参照してください。",
+      "chrome" : "Chrome",
+      "what-not-supported" : "{{what}}はサポートされていません",
+      "suggest-chrome-message" : "しばらくは、{{chromeLink}}または、Chromiumベースのブラウザを使用することをおすすめします。",
+      "start-on-boot" : "システム起動時にRainwayを起動する",
+      "start-on-boot-description" : "システムを起動するとすぐにRainwayが起動します。",
+      "minimize-on-start" : "起動時に最小化する",
+      "minimize-on-start-description" : "トレイ内でRainwayを最小化して開始します",
+      "minimize-to-tray" : "トレイに最小化",
+      "minimize-to-tray-description" : "Rainwayを閉じると、トレイに最小化されます。",
+      "configuring-rainway" : "Rainwayの構成",
+      "just-a-moment-message" : "準備しますので少々お待ちください...",
+      "offline-status" : "オフライン",
+      "busy-status" : "取り込み中",
+      "away-status" : "退席中",
+      "in-game-status" : "ゲーム中",
+      "online-status" : "オンライン",
+      "invisible-status" : "オンライン状態を隠す",
+      "network-settings-tab" : "ネットワーク",
+      "windows-settings-settings-tab" : "Windows設定",
+      "network-unavailable" : "ネットワークを利用できません",
+      "friends-unavailable" : "フレンドを利用できません",
+      "ready-to-host" : "ホストの準備完了",
+      "port-443-in-use" : "ポート433は{{name}}によって使用されています",
+      "port-40136 -in-use" : "ポート433は{{name}}によって使用されています",
```

Comment:
Oops, we should fix these two localization keys. It should really be
```
"port-in-use": "ポート{{port}}は{{name}}によって使用されています",
```
I'll make that change on our end. ^^

Reply:
Oh, that's a terrible mistake!
I revised it just in case.